app/frontend_messages.py

```diff
@@ -0,0 +1,20 @@
+import logging
+
+from flask.ext.babel import gettext as _
+
+logger = logging.getLogger(__name__)
+
+MESSAGES = {
+            'multiple-surveys': {
+                'TITLE': _("Information"),
+                'MSG': _("Unfortunately you can only complete one survey at a time."),
+                'INSTRUCTIONS': _("Close this window to continue with your current survey."),
+                },
```

Comment:
I think that the keys should be lower cased and `MSG` should be changed to `message`